code/game/objects/effects/decals/cleanable/food.dm

```diff
@@ -35,6 +35,11 @@
 		while(L.loc == src.loc)
 			L.adjustFireLoss(2, TRUE)
 			sleep(20)
+	if(is_species(H, /datum/species/squid))
```

Comment:
All of these `H`s need to be `L`s. `H` doesn't magically equal a human. Generally, you need to either use the var you have available or create a var for the human typepath and set it to the var you have available.

Reply:
Noted. Didn't see that define earlier.